packages/backend/src/graphql/resolvers/walletAddressAdditionalProperties.ts

```diff
@@ -1,33 +1,31 @@
 import {
   ResolversTypes,
   AdditionalProperty,
-  QueryResolvers
+  WalletAddressResolvers
 } from '../generated/graphql'
 import { ApolloContext } from '../../app'
 import { WalletAddressAdditionalProperty } from '../../open_payments/wallet_address/additional_property/model'
 
-export const getWalletAddressAdditionalProperties: QueryResolvers<ApolloContext>['additionalProperties'] =
+export const getWalletAddressAdditionalProperties: WalletAddressResolvers<ApolloContext>['additionalProperties'] =
   async (
     parent,
     args,
     ctx
-  ): Promise<ResolversTypes['AdditionalPropertyConnection']> => {
-    if (!args.walletAddressId) {
+  ): Promise<Array<ResolversTypes['AdditionalProperty']>> => {
+    const { id: walletAddressId } = parent
```

Comment:
Since its under the `WalletAddress` resolver, we can get the id off the parent (the wallet address) instead of passing it in as an arg.